src/main/java/org/openrewrite/java/spring/boot2/ConvertToSecurityDslVisitor.java

```diff
@@ -137,17 +155,20 @@ private Optional<JavaType.Method> findDesiredReplacement(J.MethodInvocation m) {
                 .filter(availableMethod -> availableMethod.getName().equals(m.getSimpleName()) &&
                         availableMethod.getParameterTypes().size() == 1 &&
                         availableMethod.getParameterTypes().get(0) instanceof JavaType.FullyQualified &&
-                        "org.springframework.security.config.Customizer".equals(((JavaType.FullyQualified) availableMethod.getParameterTypes().get(0)).getFullyQualifiedName()))
+                        FQN_CUSTOMIZER.equals(((JavaType.FullyQualified) availableMethod.getParameterTypes().get(0)).getFullyQualifiedName()))
                 .findFirst();
     }
 
-    public boolean isApplicableTopLevelMethodInvocation(J.MethodInvocation m) {
```

Comment:
this method was unused (and sneaking by IDEA's "unused" inspection by being recursive)

Reply:
I made this method for myself to use in the IDE integration. Can you please keep it? Perhaps a comment that this method is used in STS might be helpful

Reply:
done